tasks/MessageDecoder.hpp

```diff
@@ -0,0 +1,74 @@
+#ifndef COMMS_WEBRTC_MESSAGE_DECODER_HPP
+#define COMMS_WEBRTC_MESSAGE_DECODER_HPP
+
+#include "json/json.h"
+#include "stdio.h"
+#include "string.h"
+#include "memory"
```

Comment:
I don't see anything in there that would require these headers... Sure about their need ?